.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,49 @@
+name: "Windows build"
+
+on:
+  push:
+    paths-ignore:
+      - 'msvc*/**'
+      - 'check/**'
+      - 'doc/**'
+      - 'packaging/**'
+      - 'snap/**'
+      - '*.md'
+  pull_request:
+    paths-ignore:
+      - 'msvc*/**'
+      - 'check/**'
+      - 'doc/**'
+      - 'packaging/**'
+      - 'snap/**'
+      - '*.md'
+
+jobs:
+  build-windows-cmake:
+    name: Build FreeOrion on Windows
+    runs-on: windows-2019
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Download SDK
+        id: download-sdk
+        uses: suisei-cn/actions-download-file@v1
+        with:
+          url: https://github.com/freeorion/freeorion-sdk/releases/download/v11/FreeOrionSDK_11_MSVC-v141-Win32.zip
```

Comment:
Should we cache it?  https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows  ()

Reply:
Hard to say if Github Cache quicker than Github Releases file. It better to have something like ccache for MSVS.